packages/manager/apps/pci-ai-endpoints/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -1,31 +1,59 @@
-import React from 'react';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
 import { OsdsBreadcrumb } from '@ovhcloud/ods-components/react';
-import { useParams } from 'react-router-dom';
+import { useProjectUrl } from '@ovh-ux/manager-react-components';
 import { useProject } from '@ovh-ux/manager-pci-common';
-import {
-  usePciBreadcrumb,
-  BreadcrumbItem,
-} from '@/hooks/breadcrumb/useBreadcrumb';
-import appConfig from '@/pci-ai-endpoints.config';
+import { useLocation } from 'react-router-dom';
+import { BreadcrumbItem } from '@/hooks/breadcrumb/useBreadcrumb.hook';
 
 export interface BreadcrumbProps {
   customRootLabel?: string;
   appName?: string;
   items?: BreadcrumbItem[];
 }
 
-function Breadcrumb({ customRootLabel }: BreadcrumbProps): JSX.Element {
-  const label = customRootLabel || appConfig.rootLabel;
+const capitalize = (name: string) => {
+  if (!name) return '';
+  return name.charAt(0).toUpperCase() + name.slice(1).toLowerCase();
+};
 
-  const { projectId } = useParams();
+function Breadcrumb() {
+  const hrefProject = useProjectUrl('public-cloud');
+  const { t } = useTranslation('metric');
   const { data: project } = useProject();
+  const location = useLocation();
 
-  const breadcrumbPci = usePciBreadcrumb({
-    projectId,
-    appName: 'AI-Endpoints',
-  });
+  const path = useMemo(() => {
+    const pathSegments = location.pathname.split('/').filter(Boolean);
 
-  return <OsdsBreadcrumb items={breadcrumbPci} />;
+    switch (pathSegments[5]) {
+      case 'metrics':
+        return 'metrics';
+      case 'dashboard':
+        return 'dashboard';
+      default:
+        return '';
+    }
```

Comment:
I am not sure it's easy to maintain. It's possible have more resilient Breadcrumb as: `https://github.com/ovh/manager/blob/master/packages/manager/apps/hpc-vmware-managed-vcd/src/hooks/breadcrumb/useBreadcrumb.tsx` ?